README.md

```diff
@@ -2,10 +2,12 @@
 
 [![Build Status](https://github.com/rust-osdev/uart_16550/workflows/Build/badge.svg)](https://github.com/rust-osdev/uart_16550/actions?query=workflow%3ABuild) [![Docs.rs Badge](https://docs.rs/uart_16550/badge.svg)](https://docs.rs/uart_16550/)
 
-Minimal support for uart_16550 serial I/O.
+Minimal support for uart_16550 serial and memory mapped I/O.
 
 ## Usage
 
+### With `port_{stable, nightly}` feature
+
```

Comment:
These features don't exist anymore (they are mentioned in other places as well). This also applies to the `mmio` features.

Reply:
Fix

Reply:
That's all? @phil-opp